extensions/token-vault.yaml

```diff
@@ -1,14 +1,14 @@
 ---
 id: org.standardnotes.token-vault
```

Comment:
+premium extension

Reply:
Token vault is already included in your original repo, I just updated the github link.